src/scenes/Projects/List/ProjectList.tsx

```diff
@@ -60,22 +97,53 @@ export const ProjectList: FC = () => {
           </FilterButtonDialog>
         </Grid>
       </Grid>
-      <Typography variant="h3" paragraph>
-        {data ? (
-          `${formatNumber(data.projects.total)} Projects`
-        ) : (
-          <Skeleton width="12ch" />
-        )}
-      </Typography>
-      <ListContainer>
-        {listOrPlaceholders(data?.projects.items, 5).map((item, index) => (
-          <ProjectListItemCard
-            key={item?.id ?? index}
-            project={item}
-            className={classes.projectItem}
-          />
-        ))}
-      </ListContainer>
+      <TabContext value={activeTab}>
+        <Tabs
+          value={activeTab}
+          onChange={(_e, newActiveTab) => setActiveTab(newActiveTab)}
+          aria-label="project navigation tabs"
+        >
+          <Tab label="My Projects" key="mine" value="mine" />
+          <Tab label="All" key="all" value="all" />
+          <Tab label="Recent" key="recent" value="recent" />
```

Comment:
Recent was only in designs. Don't add it here as we don't have an implementation.